### PR TITLE
Link to translations.

### PR DESCRIPTION
**Pull request by chriseth**

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -20,6 +20,16 @@ crowdfunding, blind auctions, multi-signature wallets and more.
     `Remix <https://remix.ethereum.org/>`_
     (it can take a while to load, please be patient).
 
+Translations
+------------
+
+This documentation is translated into several languages, of which the English
```
Comment (axic) [inline docs/index.rst]: This suggests English is a translation too, but doesn't clarifies what is the original: "translated into several languages, of which the English translation"

Comment (chriseth) [reply docs/index.rst]: the original is an abstract concept without language ;-)

Comment (chriseth) [reply docs/index.rst]: We could say "is available in several languages, of which the English version is ..."?

Comment (axic) [reply docs/index.rst]: How about:

"This documentation is translated into several languages, but likely the English original is the most up date."

or 

"This documentation is translated into several languages by third party volunteers, but the this English version stands as a reference."

**Review by axic — APPROVED** [review]

**Pull request merged**